File: Java/EvenOdd.java
import java.util.*;
public class EvenOdd

{
    public static void main(String args[])
    {
        System.out.println("To check whether a number is even or odd");

        Scanner sc=new Scanner(System.in);
        System.out.println("Enter a number:");
        int n=sc.nextInt();

        if(n%2==0)
        {
            System.out.println(n+" is an even number");
        }
        else
        {
            System.out.println(n+" is an odd number");
        }
    }


    
}

File: C#/EvenOdd.cs
using System;
class EvenOdd
{
    static void Main(string[] args)
    {
        Console.WriteLine("Enter a Number:");
        int n=Convert.ToInt32(Console.ReadLine());

        if(n%2==0)
        {
            Console.WriteLine("The Number is Even");
        }
        else
        {
            Console.WriteLine("The Number is Odd");
        }

        Console.ReadKey();
    }
}
